packages/cogutil/cogutil.opam-1.0/opam

```diff
@@ -0,0 +1,29 @@
+(** OPAM (OCaml Package Manager) module for CogUtil *)
+(** *)
+(** This is in the root directory because I cannot figure out *)
+(** how to tell opam to look for it somewhere else. Sigh. *)
+(** *)
+(** To build, say: `sudo ls -la` and provide your password. *)
+(** Then say `opam install .` in the root git directory. *)
+(** I have no clue how to get this to build without the sudo step. *)
+(** Finally, say `eval $(opam env)` *)
+
+opam-version: "2.0"
+synopsis: "OpenCog CogUtil base package"
+description: """
+The CogUtil package provides base infrastructure for other OpenCog
+packages. This includes scripts for building OCaml bindings for
+OpenCog modules.
+"""
+maintainer: "linasvepstas@gmail.com"
+authors: "Linas Vepstas"
+license: "AGPL-3.0-or-later"
+homepage: "https://opencog.org"
+bug-reports: "https://github.com/opencog/cogutil/issues"
+dev-repo: "git+https://github.com/opencog/cogutil/"
+depends: [ "g++" "cmake" "libboost-dev" "libboost-program-options-dev" "libboost-system-dev" "libboost-thread-dev" "binutils-dev" "libiberty-dev" "ocaml" "ocamlfind" ]
+build: [
+  ["cmake" "."]
+  [make]
+]
+install: ["sudo" "make" "install"]
```

Comment:
You cannot run sudo in the opam sandbox, commands cannot access any folder outside the sources folder